src/templates/plugin.jsx

```diff
@@ -95,6 +99,7 @@ function PluginPage({data: {jenkinsPlugin: plugin, reverseDependencies: reverseD
                                 installations={plugin.stats.installations}
                             />
                         </div>
+                        <div className="label-link"><a href={`https://stats.jenkins.io/pluginversions/${plugin.name}.html`}>Installs by version</a></div>
```

Comment:
```suggestion
                        <div className="label-link"><a href={`https://stats.jenkins.io/pluginversions/${plugin.name}.html`}>View detailed version information</a></div>
```

It's not just _plugin_ versions that are installed, also _core_ -- that may actually be the more important part anyway.